### ISSUE TITLE
Via fails for Blogger URLs - http://blog.raintown.org/2015/08/wanted-programming-models-for-cloud.html

### ISSUE THREAD

**Issue by robertknight** [hypothesis/via#68]
The sidebar fails to load on this page: https://via.hypothes.is/http://blog.raintown.org/2015/08/wanted-programming-models-for-cloud.html

**Comment by ikreymer** [comment]
Yeah, Blogger is one of those sites that wipes the DOM on load.. also removes any pywb banner, so would need some detection approach for these types of sites.
